commands/docker/docker-clean.md

```diff
@@ -0,0 +1,41 @@
+### Cleaning up docker to reclaim disk space
+
+When the disk space was "leaking” somewhere and you're not able to tell where it went,
+clean the unused storage of the docker using this command :
+
+`docker system prune`
+
+#### Example:
+
+`docker system prune` <br>
+                     
+WARNING! This will remove:<br>
```

Comment:
We don't want to add logs like this.
If you want to add working logs, we would love to have a gif.
Other wise just the example command is fine